Description: Hardcode python path
 There is no python3-gps package yet.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2016-12-04
--- a/contrib/gps-log
+++ b/contrib/gps-log
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 # coding: utf-8
 
 from __future__ import print_function
